com.unity.render-pipelines.high-definition/Editor/Wizard/HDWizard.Window.cs

```diff
@@ -17,12 +18,6 @@ static class Style
 
             public const string hdrpProjectSettingsPathLabel = "Default Resources Folder";
```

Comment:
note for future us: make sure those string can be localized

Reply:
Yes sorry it is the wold wizard. It should be a PR in itself as some label need to be extracted in a style class.